django/core/management/base.py

```diff
@@ -672,7 +672,16 @@ class LabelCommand(BaseCommand):
     """
 
     label = "label"
-    missing_args_message = "Enter at least one %s." % label
+    missing_args_message = "Enter at least one %s."
+
+    def __init__(self, *args, **kwargs):
+        super().__init__(*args, **kwargs)
+
+        if (
+            LabelCommand.missing_args_message == self.missing_args_message
+            or self.missing_args_message is None
+        ):
```

Comment:
```suggestion
        if self.missing_args_message == LabelCommand.missing_args_message:
```
I think when `missing_args_message = None` we shouldn't have an error message, which means you can update the test `test_custom_label_command_none_missing_args_message` to `with self.assertRaisesMessage(CommandError, ""):`

Reply:
My bad, for some reason I thought that setting `missing_args_message` to `None` should make us fall back to some default message. Thanks for the help and the patience!

Reply:
Thank you for the continued updates!